doc/md/guides/postgres/included-columns.md

```diff
@@ -0,0 +1,316 @@
+   Rows Removed by Filter: 1499
+ Planning Time: 0.053 ms
+ Execution Time: 0.195 ms
+(5 rows)
+
+Time: 0.626 ms
+```
+
+Notice that the total cost is 38.75 units. If we want to use a unique index to accelerate the query, we can create it on the “email” column with the following command:
+
+```sql
+CREATE UNIQUE INDEX emails_idx 
+ON bankdb(email);
+```
+```console title="Output"
+CREATE INDEX
+Time: 10.316 ms
+```
+
+Now, let's check the performance of querying data of first and last names based on their email addresses, with the following command:
+
+```sql
+EXPLAIN ANALYZE
+SELECT    
+    first_name,
+    last_name,
+    email
+FROM    
+    "bankdb"
+WHERE 
+    email = 'd-abbott3425@google.edu';
+```
+```console title="Output"
+QUERY PLAN                   
+----------------------------------
+ Index Scan using emails_idx on bankdb  (cost=0.28..8.29 rows=1 width=37) (actual 
+time=0.200..0.203 rows=1 loops=1)
+   Index Cond: ((email)::text = 'd-abbott3425@google.edu'::text)
+ Planning Time: 0.081 ms
+ Execution Time: 0.259 ms
+(4 rows)
+
+Time: 1.470 ms
+```
+
+Notice that total cost is now 8.29 units. The performance of the query has improved by creating a primary key index on “email” column, compared to 38.75 units without using any index. The engine still has to fetch the “first_name” and “last_name” columns from the table (also known as “Heap fetches”).
+Let's drop the existing index to demonstrate next section in the article
+
+```sql
+DROP INDEX emails_idx;
+```
+```console title="Output"
+DROP INDEX
+Time: 3.856 ms
+```
+
+Suppose we want to accelerate the same query using the INCLUDE clause.
+In the following command, we have created an index with an INCLUDE clause that precisely covers “first_name” and “last_name” columns which are part of the query for which we are trying to improve performance.
+
+```sql
+CREATE UNIQUE INDEX emails_idx 
+ON bankdb(email)
+INCLUDE(first_name,last_name);
+```
+
+```console title="Output"
+CREATE INDEX
+Time: 8.942 ms
+```
+
+Now, let's check the performance of querying data of first and last names based on their email addresses, with the following command:
+
+```sql
+EXPLAIN ANALYZE
+SELECT    
+    first_name,
+    last_name,
+    email
+FROM    
+    "bankdb"
+WHERE 
+    email = 'd-abbott3425@google.edu';
+```
+```console title="Output"
+QUERY PLAN                 
+---------------------------------------
+ Index Only Scan using emails_idx on bankdb  (cost=0.28..4.29 rows=1 width=37) (ac
+tual time=0.228..0.231 rows=1 loops=1)
+   Index Cond: (email = 'd-abbott3425@google.edu'::text)
+   Heap Fetches: 0
+ Planning Time: 0.233 ms
+ Execution Time: 0.283 ms
+(5 rows)
+```
+Notice that the total cost is now 4.29, which is significantly lower compared to 8.29 while using a unique index without the INCLUDE clause. Because, the query only scanned the index in order to get the data. As a result, “Heap fetches” is also zero, which means the query does not access any tables to retrieve the records. 
```

Comment:
```suggestion
Notice that the total cost is now 4.29, which is significantly lower, compared to 8.29 which we got while using a unique index without the `INCLUDE` clause. We were able to reduce the total cost because the query only scanned the index in order to get the data. As a result, `heap fetches` is also zero, which means the query does not access any tables to retrieve the records. 
```